### ISSUE TITLE
auto documenting buildable assets

### ISSUE THREAD

**Issue by stolarczyk** [refgenie/refgenie#239]
It would be nice to generate some sort of [this page](http://refgenie.databio.org/en/latest/available_assets/) automatically, based on the set of available recipes. This will come in handy once the recipes are no longer a part of refgenie package, but are served with refgenieserver.

It should be relatively easy to produce at least a minimal markdown entry for each asset in case if only the keys suggested in [this issue](https://github.com/refgenie/refgenie/issues/198#issue-683071342) are defined. But we could also provide means to add more detailed documentation for each recipe. For example, `docs` key that is a freeform text, later interpreted as markdown, or something.

**Comment by nsheff** [comment]
Yeah that would be great! This is along the lines of a recipe splash page.
